test/testdata/expected/EmptyStruct.go

```diff
@@ -0,0 +1,7 @@
+package expected
```

Comment:
It is a Go convention to name Go files with lowercase and underlines, so this file should be called "empty_struct.go"